### PR TITLE
docs: switch to orama

### LINKED ISSUE #4396: Website - search is broken
## Which package?

website

## Describe the bug

In the last couple of weeks, the search functionality is broken.
Each time that you search for something, in the console is logged the following message:
```Invalid Application-ID or API key```

## To Reproduce

Search for anything in the website

### Codesandbox URL

Please include a codesandbox URL if possible

## Expected behavior

I can search what I need

## Screenshots

image

## Additional context

Suggestion: to avoid this also in the future, what if you switch to [Orama](https://askorama.ai/) by @micheleriva?

It's open source and doesn't require any api key.

### PR DESCRIPTION
**Pull request by ilteoood**
Closes #4396

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Pull requests are very welcome, but should be within the scope of the project, a
 - [`bazel`](https://bazel.build/)
 - [`docker`](https://www.docker.com/)
 
-You can build & test with `pnpm`:
+You can build & test with `pnpm`. At the moment version >= 9 is not supported:
```
Comment (ilteoood) [inline CONTRIBUTING.md:36]: pnpm version 9 brings package lock to version 9, which seems to be unsupported by the bazel plugin used here.

File: website/package.json [hunk 2]
```diff
@@ -15,15 +15,15 @@
...
   },
   "devDependencies": {
-    "@algolia/client-search": "4",
-    "@docusaurus/core": "3",
-    "@docusaurus/plugin-google-analytics": "3",
-    "@docusaurus/preset-classic": "3",
-    "@docusaurus/theme-common": "3",
-    "@docusaurus/theme-live-codeblock": "3",
-    "@docusaurus/types": "3",
+    "@docusaurus/core": "3.2.1",
```
Comment (ilteoood) [inline website/package.json:18]: Per doc, orama requires at least version 3.2.0 of docusaurus

**Comment by longlho** [comment]
Thanks a lot for your contribution I've manually merged this in

**Comment by ilteoood** [comment]
@longlho will you close the issue when it'll be released?

**Comment by longlho** [comment]
It's released but still broken due to some server hydration issue from docusaurus...

**Comment by micheleriva** [comment]
Hi @longlho, I'm one of the Orama founders. I see this is not working in prod, let me ask my colleague @raiindev to look into this.

Alternatively, @ilteoood we can open a bug in the Orama repo and treat this as a bounty

**Pull request closed without merging**